client/src/js/components/bhRequisitionSelect.js

```diff
@@ -31,9 +32,14 @@ function RequisitionSelectController(Stock, Notify) {
     const params = {};
     params.requestor_uuid = $ctrl.requestorUuid;
 
+    $ctrl.allowCompleted = false;
+
     Stock.stockRequisition.read(null, params)
       .then((requisitions) => {
-        $ctrl.requisitions = requisitions;
+        const allowCompleted = $ctrl.allowCompleted !== 'false';
```

Comment:
Why is `false` a boolean in line 35 and a string here?

Can we coerce one into the other?

Reply:
Changed the logic on the flag to clean this up.